.github/workflows/publish.yml

```diff
@@ -28,20 +21,15 @@ jobs:
       - name: Install dependencies
         run: bash ./install_deps.sh
 
-      - name: Compile Project
+      - name: Compile and prepare package
```

Comment:
Now we are merging `Release build` and `Generate Source jar` in our `Compile and prepare package`, making it a single command instead of multiple commands of `./gradlew` would be better. Gradle will automatically execute all tasks one by one.

Reply:
Now it is done and LGTM.